silkworm/db/state/receipts_domain.hpp

```diff
@@ -50,8 +50,9 @@ static_assert(snapshots::DecoderConcept<ReceiptsDomainKeySnapshotsDecoder>);
 struct VarintSnapshotsDecoder : public snapshots::Decoder {
     uint64_t value{};
     ~VarintSnapshotsDecoder() override = default;
-    void decode_word(ByteView word) override {
-        auto value_opt = snapshots::seg::varint::decode(word);
+    void decode_word(Bytes& word) override {
+        ByteView word_view{word};
+        auto value_opt = snapshots::seg::varint::decode(word_view);
```

Comment:
```suggestion
        auto value_opt = snapshots::seg::varint::decode(word);
```
will implicit conversion work?